MaxText/layers/linears.py

```diff
@@ -294,8 +296,8 @@ def generate_kernels(self, num_experts, emb_dim, mlp_dim):
     kernel_out_axis = np.arange(1, 2)
     kernel_init = nd_dense_init(1.0, 'fan_in', 'truncated_normal')
 
-    kernel_axes = ('exp', 'embed', 'mlp')
-    wo_kernel_axes = ('exp', 'mlp', 'embed')
+    kernel_axes = (None, 'embed', 'mlp')
```

Comment:
Nit: Add a comment here that the first axes corresponds to experts

Reply:
Sg! Will add that.